jax/test_util.py

```diff
@@ -14,6 +14,7 @@
 
 
 from contextlib import contextmanager
+from distutils.util import strtobool
```

Comment:
It feels like this is an inappropriate dependency. While `distutils` is in the Python standard library, this is a very strange reason to use it.

(I suspect this is probably copy-paste from another flag, but just an observation.)

Reply:
I added its use in other files. My understanding is that this is precisely its intended use case. Is there an alternative?

Reply:
Yes: don't take the dependency, implement it ourselves in, say, config.py. It's probably only ~10 lines of code. It's very odd to depend on what seems to be an implementation detail of the Python module distribution library that probably shouldn't have been a public API in the first place.

Reply:
I did this, added config.bool_env, which is actually better.